### PR TITLE
Fix owner problems in 2.2.0+

### PR DESCRIPTION
**Pull request by offirgolan**
In reference to #86

### CODE REVIEW

**Comment by offirgolan** [comment]
@rwjblue @stefanpenner can I get a quick review on this. This is a temporary fix until a setOwner polyfill is created. Also, ignore the build failing, its due to components/ember-data#34

File: app/validators/date.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 import Ember from 'ember';
 import Base from 'ember-cp-validations/validators/base';
-const moment = self.requirejs.entries['moment'] && self.require('moment')['default'];
+const moment = (self.requirejs.entries['moment'] || self.requirejs.entries['moment/index']) && self.require('moment')['default'];
```
Comment (rwjblue) [inline app/validators/date.js]: We should add a nice `loaderjs.has` helper method for this, but that is another issue altogether :smiley_cat:

**Comment by rwjblue** [comment]
@offirgolan - This looks like a good workaround for now, sorry I've been so delayed in adding support upstream.

File: app/validators/date.js (hunk 1 above)
Comment (offirgolan) [inline app/validators/date.js]: That would definitely be nice :smile:

**Comment by offirgolan** [comment]
@rwjblue no rush! This should hold down the fort for a while

**Pull request merged**